source/jormungandr/jormungandr/scenarios/experimental.py

```diff
@@ -118,7 +135,7 @@ def _update_crowfly_duration(instance, mode, stop_area_uri, requested_point):
         for stop_point in odt_sps:
             fallback_list[mode][stop_point.uri] = 0
             odt_stop_points.add(stop_point.uri)
-    
+
     return crowfly_sps, odt_stop_points, fallback_list
```

Comment:
https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/scenarios/experimental.py#L502

That's the same format as `g.*_places_crowfly`